### ISSUE TITLE
Package has not been updated for R Version 3.6.1

### ISSUE THREAD

**Issue by agcfzk** [USGS-R/streamMetabolizer#407]
### Brief problem description

I'm trying to run the package on R but when installing the following warning appears "package ‘streammetabolizer’ is not available (for R version 3.6.1)".  Are the contributors planning to update the package?

```r

install.packages(StreamMetabolizer)

```

```r
Warning in install.packages :
  package ‘streammetabolizer’ is not available (for R version 3.6.1)

```

**Comment by robohall** [comment]
You may need to install from Github using the procedure described here:
http://usgs-r.github.io/streamMetabolizer/articles/installation.html

Bob

On May 20, 2021, at 2:09 PM, agcfzk ***@***.******@***.***>> wrote:


Brief problem description

I'm trying to run the package on R but when installing the following warning appears "package ‘streammetabolizer’ is not available (for R version 3.6.1)". Are the contributors planning to update the package?

install.packages(StreamMetabolizer)


Warning in install.packages :
  package ‘streammetabolizer’ is not available (for R version 3.6.1)

**Comment by aappling-usgs** [comment]
Yes, use the instructions at the link Bob shared. Also, pay attention to capitalization - it's `streamMetabolizer`, not `StreamMetabolizer` or `streammetabolizer`, and the code will care.
